src/plugins/extra/cite/lib/cite-plugin.js

```diff
@@ -754,8 +807,8 @@ define([
 						$elem.removeAttr('data-cite-id');
 					}
 				}
-				$elem.removeClass('aloha-cite-wrapper');
```

Comment:
this looks like it might be a breaking change ... why are we removing it?

Reply:
Because of loading an already existing citations. This class has been used for marking only just added citations, and is erased on editor close. Since we want to load citations back next time we use the cite editor, we have to keep some mark for the editor.
https://github.com/bernhardriegler/Aloha-Editor/pull/1/files#diff-a334bfb884f2a016f73ac4d0d0df1c22R505

Reply:
fair